developers/bindings/thing-xml.md

```diff
@@ -0,0 +1,717 @@
+
+The `advanced` property indicates whether this channel is a basic or a more specific functionality of the thing.
+If `advanced` is set to `true` a user interface may hide this channel by default.
+The default value is `false` and thus will be taken if the `advanced` attribute is not specified.
+Especially for complex devices with a lot of channels, only a small set of channels - the most important ones - should be shown to the user to reduce complexity.
+Whether a channel should be declared as `advanced` depends on the device and can be decided by the binding developer.
+If a functionality is rarely used it should be better marked as `advanced`.
+
+The following XML snippet shows a trigger channel:
+
+```xml
+<thing-type id="thingTypeID">
+    <label>Sample Thing</label>
+    <description>Some sample description</description>
+    <channels>
+        <channel id="s" typeId="trigger-channel" />
+    </channels>
+</thing-type>
+<channel-type id="trigger-channel">
+    <kind>trigger</kind>
+    <label>Trigger Channel</label>
+    <event>
+        <options>
+            <option value="PRESSED">pressed</option>
+            <option value="RELEASED">released</option>
+            <option value="DOUBLE_PRESSED">double pressed</option>
+        </options>
+    </event>
+</channel-type>
+```
+
+This channel can emit the event payloads `PRESSED`, `RELEASED` and `DOUBLE_PRESSED`.
+
+If no `<event>` tag is specified, the channel can be triggered, but has no event payload.
+If an empty `<event>` tag is specified, the channel can trigger any event payload.
+
+### System Trigger Channel Types
+
+There exist system-wide trigger channel types that are available by default:
+
+| Channel Type ID | Reference typeId       | Description  |
+|-----------------|------------------------|------------- |
+| trigger         | system.trigger         | Can only trigger, no event payload |
+| rawbutton       | system.rawbutton       | Can trigger `PRESSED` and `RELEASED` |
+| button          | system.button          | Can trigger `SHORT_PRESSED`, `DOUBLE_PRESSED` and `LONG_PRESSED` |
+| rawrocker       | system.rawrocker       | Can trigger `DIR1_PRESSED`, `DIR1_RELEASED`, `DIR2_PRESSED` and `DIR2_RELEASED` |
+
+In the following sections the declaration and semantics of tags, state descriptions and channel categories will be explained in more detail. 
+For a complete sample of the thing types XML file and a full list of possible configuration options please see the [XML Configuration Guide](xml-reference.html).
+
+### Default Tags
+
+The XML definition of a ThingType allows to assign default tags to channels. 
+All items bound to this channel will automatically be tagged with these default tags. 
+The following snippet shows a 'Lighting' tag definition:
+
+```xml
+<tags>
+    <tag>Lighting</tag>
+</tags>
+```
+
+Please note that only tags from a pre-defined tag library should be used.
+This library is still t.b.d., and only a very small set of tags are defined so far:
```

Comment:
What is the intended "t.b.d." here? to be discussed? to be defined? "in active development"?

Reply:
No idea, ask the ESH docu writters. But you are also invited to come up with a better sentence :)